packages/headless/src/controllers/index.ts

```diff
@@ -27,6 +27,8 @@ export {
   buildContext,
 } from './context/headless-context';
 
+export {buildDebug} from './debug/headless-debug';
```

Comment:
You should export the interface and types too: DebugState & Debug

Reply:
Yes indeed.

Reply:
@ThibodeauJF I think exporting DebugState does not apply anymore, since I'm now exporting only actions.